.github/ISSUE_TEMPLATE/new_issue_template.yml

```diff
@@ -0,0 +1,63 @@
+name: SS3 Team Issues - New Template
+description: Issue template for SS3 team members ONLY
+title: "[Issue]: "
+assignees:
+  - Rick-Methot-NOAA
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this issue!
+- type: textarea
+  id: describe
+  attributes:
+    label: Issue Description
+    description: Please provide a description of the issue.
+  validations:
+    required: true
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary of Issue History, Discussion, and Major Aspects of Code Development 
+    description: A summary of pertinent information on this issue, including major aspects of code development, that occurred in the discussion/via email. The person creating the issue is responsible for adding info from the discussion to this section.
+    placeholder: |
+      - This issue is realted to issue #X.
+      - Will need to create an example vignette for this feature.
+      - example input change
+        ```
+        #_MG_type method st_year end_year 
+        1 1 2002 2003 # (M) NatMort 
+        4 1 2016 2018 # RecrDist 
+        -9999 -1 -1 -1
+        ```
+  validations:
+    required: true
+- type: textarea
+  id: tasks
+  attributes:
+    label: Items to do
+    description: Please create a list of tasks to be completed for this issue with the checkboxes.
+    value: "- [ ] First Item"
+  validations:
+    required: true
+- type: checkboxes 
+  id: documentation
+  attributes:
+    label: Does documentation already exist?
+    description: Please check one of the boxes.
+    options:
```

Comment:
I added the link to the user manual here and reworded this a bit. I didn't include the "i'm not sure" option because my hope is that people will look for the documentation before answering. However, I can always add the "i'm not sure" option if people think they will need it.